samples/openapi3/client/petstore/go-experimental/go-petstore/api/openapi.yaml

```diff
@@ -54,7 +54,7 @@ paths:
       requestBody:
         $ref: '#/components/requestBodies/Pet'
       responses:
-        405:
+        "405":
```

Comment:
Why is this file in this update?
Aren't we updating java/python/python-experimental here only?

Reply:
I think this is because of some unrelated problem. I see other PRs opened recently have the same changes..

Reply:
I have opened a separate PR for this one: #5049. It looks like somebody forgot to execute a script under ./bin